projects/Rockchip/bootloader/release

```diff
@@ -1,23 +1,23 @@
 # SPDX-License-Identifier: GPL-2.0
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
+SRCDIR="$BUILD/image/system/usr/share/bootloader"
+
 mkdir -p $RELEASE_DIR/3rdparty/bootloader
   if [ -n "$UBOOT_SYSTEM" ]; then
-    BOOTLOADER_DIR=$(get_build_dir $BOOTLOADER)
-    if [ -f $BOOTLOADER_DIR/idbloader.img ]; then
-      cp -a $BOOTLOADER_DIR/idbloader.img $RELEASE_DIR/3rdparty/bootloader
+    if [ -f $SRCDIR/idbloader.img ]; then
+      cp -a $SRCDIR/idbloader.img $RELEASE_DIR/3rdparty/bootloader
     fi
-    if [ -f $BOOTLOADER_DIR/uboot.img ]; then
-      cp -a $BOOTLOADER_DIR/uboot.img $RELEASE_DIR/3rdparty/bootloader
+    if [ -f $SRCDIR/uboot.img ]; then
+      cp -a $SRCDIR/uboot.img $RELEASE_DIR/3rdparty/bootloader
     fi
-    if [ -f $BOOTLOADER_DIR/trust.img ]; then
-      cp -a $BOOTLOADER_DIR/trust.img $RELEASE_DIR/3rdparty/bootloader
+    if [ -f $SRCDIR/trust.img ]; then
+      cp -a $SRCDIR/trust.img $RELEASE_DIR/3rdparty/bootloader
     fi
   fi
 
   DTB_PREFIX="${DEVICE/RK/rk}-"
-  LINUX_DTS_DIR=$(get_build_dir linux)/arch/$TARGET_KERNEL_ARCH/boot/dts
-  for dtb in $LINUX_DTS_DIR/$DTB_PREFIX*.dtb $LINUX_DTS_DIR/rockchip/$DTB_PREFIX*.dtb; do
+  for dtb in $SRCDIR/$DTB_PREFIX*.dtb $SRCDIR/rockchip/$DTB_PREFIX*.dtb; do
```

Comment:
We should be able to replace this entire for block with:
`cp -a $SRCDIR/${DEVICE/RK/rk}-*.dtb $RELEASE_DIR/3rdparty/bootloader`

Reply:
done